.github/workflows/wails.yml

```diff
@@ -31,7 +31,6 @@ jobs:
       matrix:
         build:
           [
-            { platform: linux/amd64, os: ubuntu-20.04 },
```

Comment:
this removes the Ubuntu 22.04 Wails build, can you make sure this line is left in but change it to 22.04?

Reply:
Done!